core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -452,16 +452,23 @@ public void removeBlocks(List<Long> blockIds, boolean delete) throws Unavailable
   @Override
   public void validateBlocks(Function<Long, Boolean> validator, boolean repair)
       throws UnavailableException {
-    List<Long> invalidBlocks = new ArrayList<>();
-    for (long blockId : mBlocks.keySet()) {
-      if (!validator.apply(blockId)) {
-        invalidBlocks.add(blockId);
+    List<Long> invalidBlocks = mBlocks.keySet()
```

Comment:
Why is the format here different from the format of the following `stream`?

Reply:
fixed